lib/reverse_sentence.rb

```diff
@@ -4,22 +4,6 @@ def reverse_sentence(my_sentence)
   return reverse_words(reversed_string)
```

Comment:
Since the string_reverse reverses the input string in place, you don't need to return anything or assign anything. reverse_sentence could simply be:
```ruby
   string_reverse(my_sentence)
   reverse_words(my_sentence)
   return
```